src/inmanta/protocol/rest/server.py

```diff
@@ -140,6 +141,11 @@ async def _call(self, kwargs: Dict[str, str], http_method: str, call_config: com
                 else:
                     raise exceptions.UnauthorizedException("Access to this resource is unauthorized.")
 
+            except JSONDecodeError:
+                error_message = "The request body couldn't be decoded as a JSON"
```

Comment:
The exception also contains more details about where the errors. Should we provide that? See the original ticket for that. @wouterdb I do not see any security issues to report this.

Reply:
agreed